static/app/components/replaysOnboarding/useCurrentProjectState.tsx

```diff
@@ -1,99 +0,0 @@
-import {useEffect, useMemo, useState} from 'react';
-
-import {splitProjectsByReplaySupport} from 'sentry/components/replaysOnboarding/utils';
-import {SidebarPanelKey} from 'sentry/components/sidebar/types';
-import {replayOnboardingPlatforms, replayPlatforms} from 'sentry/data/platformCategories';
-import PageFiltersStore from 'sentry/stores/pageFiltersStore';
-import {useLegacyStore} from 'sentry/stores/useLegacyStore';
-import type {Project} from 'sentry/types';
-import useProjects from 'sentry/utils/useProjects';
-
-function useCurrentProjectState({currentPanel}: {currentPanel: '' | SidebarPanelKey}) {
-  const [currentProject, setCurrentProject] = useState<Project | undefined>(undefined);
-  const {projects, initiallyLoaded: projectsLoaded} = useProjects();
-  const {selection, isReady} = useLegacyStore(PageFiltersStore);
-
-  const isActive = currentPanel === SidebarPanelKey.REPLAYS_ONBOARDING;
-
-  // Projects where we have the onboarding instructions ready:
-  const projectsWithOnboarding = useMemo(
-    () =>
-      projects.filter(
-        p => p.platform && replayOnboardingPlatforms.includes(p.platform) && !p.hasReplays
```

Comment:
Does the `!p.hasReplays` go somewhere else?

Seems like the intention was so that we don't list projects in the dropdown if they already have captured a replay|ddm|feedback event, afaik

`hasReplays` is mentioned here and the next statement `const projectWithReplaySupport = ...`

Reply:
i noticed this, but also saw that DDM didn't use a similar flag in their hook - we could add a new prop here to pass in a flag and do this extra check, but not sure what behavior @ArthurKnaus was expecting for his sidebar

Reply:
In DDM the dropdown contains all the projects. Even if they have custom metrics, as we want them to be able to revisit the instructions.